### DISCUSSION TITLE
Custom Installation Paths Install Strings

### DISCUSSION THREAD

**Discussion by Poagie** [coreybutler/nvm-windows#896] [Support]
How can I tell nvm to install in a different path using installation strings? Using the user profile in enterprise is just a plain terrible default path. Would like to be able to tell the installer to install both the NVM and the symlink path to a custom location. We need this for both security purposes and ease of use. Anyone know the correct strings? I cant seem to find any information around this for Windows install.

**Comment by coreybutler** [comment]
The installer will accept custom paths. On modern versions of Windows, you can just set the environment variables. Some enterprises have created their own installers (it's actually not too tough). See https://github.com/coreybutler/nvm-windows/wiki/Hacking, which dives into the details.

We've built custom installers for clients, so there is a commercial route for those who are interested... but it's something many groups can do on their own. I'm also [actively seeking feedback]((https://t.co/oGqQCM9FPx)) like this to better understand enterprise challenges as we build [Runtime](https://github.com/coreybutler/nvm-windows/wiki/Runtime) (for enterprise).
